README.md

```diff
@@ -337,58 +338,40 @@ This example purchase of a perpetual contract of AAVE with a leverage of 20, usi
 
     { "Exchange":"ftx","Market":"Future","Account":"MAIN","Action":"Buy","Asset":"AAVE/USD:USD","Base":"1","Leverage":"20","Margin":"Isolated" }
 
-Discriptionn of the payload
+Description of the payload
 
-    Exchange
+* `Exchange`: This is one of the [supported exchanges](#exchanges).
 
-        This is one of the supported exchanges.
-
-    Market
-
-        This is the market you are trading. 
+* `Market`: This is the market you are trading. 
 
         Spot            Working
         Future          Working
         Margin          In progress
 
-    Account
-
-        All main accounts must be called MAIN. Subaccounts can be used
-        and must be exactly as listed on the exchange.
+* `Account`: All main accounts must be called MAIN. Subaccounts can be used (if your exchange supports them) and must be exactly as listed on the exchange.
 
-    Action
+* `Action`:
 
         Buy             Make a purchase
         Sell            Sell a portion. If amount is more then balance, position will be closed
         Close           Sell all of the asset
         Long            (Perpetuals) Opens a long position and can flip a short to long
         Short           (Perpetuals) Opens a short position and can flip a long position to short
 
-    Asset
-
-        The asset you are trading. Must be exactly as the exchange lists it
-
-    USD/Base/Quote
+* `Asset`: The asset you are trading. Must be exactly as the exchange lists it. Hint: Use [ListMarkets](Extras/README.md#listmarkets) to confirm
 
-        Choose only one to set the amount to be purchased/sold
-
-        USD will only work if the asset has a coresponsing USD pair to do
+* `USD`/`Base`/`Quote`: Choose only one to set the amount to be purchased/sold
+    * `USD` will only work if the asset has a corresponding USD pair to do
```

Comment:
@rapmd73 Is this right? I don't understand yet how `USD` would be used? Like, if it's an ETH/BTC pair, could you use `"USD": "35"` for a $35 equivalent buy/sell? If so, I wonder what exchange rate is used

Reply:
This is been a hot mess since day one and I really haven't figured out a better way of conveying how to convert to and from USD easily. I have some experiments I've been running and working on to merge this down to a much more efficient way.

One way that I think might work the best is just to treat USD as a standardized quote currency within the entire system.